docs/providers/aws/guide/functions.md

```diff
@@ -501,3 +501,25 @@ functions:
     maximumEventAge: 7200
     maximumRetryAttempts: 1
 ```
+
+## EFS Configuration
+
+You can use [Amazon EFS with Lambda](https://docs.aws.amazon.com/lambda/latest/dg/services-efs.html) by adding a `fileSystemConfigs` property in the function configuration in `serverless.yml`. `fileSystemConfigs` should be a list of objects that contain the `arn` and `localMountPath` properties. The `arn` property should reference an existing EFS Access Point, where the `localMountPath` should specifiy the absolute path under which the file system will be mounted. Currently, you can specify at most one `fileSystemConfig`. Here's an example configuration:
```

Comment:
At the moment it's possible to attach only one fileSystem to Lambda and I'm not sure when/if that's going to change in the future. In CloudFormation, they're specified as an array with maximum of 1 (https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-lambda-function.html#cfn-lambda-function-filesystemconfigs), most likely as a way to be future-proof when (potentially) it would be possible to attach more than one filesystem to Lambda. It might make sense in case of `serverless` to have this as singular `fileSystemConfig`, however, what do you think the migration in the future should look like if it would become possible to configure more than one filesystem?

Reply:
Let me check with AWS what's the probability of having that lifted soon (I'll get back once I have an answer.. probably in 24-48 hours timeframe)

Reply:
Thank you :bow:

Reply:
@pgrzesik I didn't receive any specific answer.

I'd say let's make it singular, so `fileSystemConfig` (not `fileSystemConfigs`), and if eventually AWS will shift the limit, we will introduce `fileSystemConfigs`, at this point I doubt it'll be the case anytime soon.